upmc-enterprises/cda2fhir
# Upmcfhir 242 expand reference object

#### What does this PR do?

Adds a "CreateReference" method that adds a display and an identifier to all references. 

I also formatted the file. So that resulted in a lot more changed lines. 

#### Related JIRA tickets:

[UPMCFHIR-242](https://jira.amida.com/browse/UPMCFHIR-242)

#### How should this be manually tested?

Run the integration test while printing the result and check that the references have identifiers and displays when expected.

#### Background/Context:

#### New JIRA tickets for clinical review required?
